SchedulerJobs/SchedulerJobs.Services/VideoApiService.cs

```diff
@@ -74,10 +88,31 @@ public async Task<List<ExpiredConferencesResponse>> GetExpiredOpenConferences()
             _log.LogTrace($"Getting expired open conferences");
 
             var uriString = _apiUriFactory.ConferenceEndpoints.GetExpiredOpenConferences();
+            return await GetListExpiredConferencesResponse(uriString);
+        }
+
+        public async Task<List<ExpiredConferencesResponse>> GetExpiredAudiorecordingConferences()
+        {
+            _log.LogTrace($"Getting expired audiorecording conferences");
+
+            var uriString = _apiUriFactory.ConferenceEndpoints.GetExpiredAudiorecordingConferences;
+            return await GetListExpiredConferencesResponse(uriString);
+        }
+
+        private async Task<List<ExpiredConferencesResponse>> GetListExpiredConferencesResponse(string uriString)
+        {
             var response = await _httpClient.GetAsync(uriString).ConfigureAwait(false);
```

Comment:
Dont need ConfigureAwait(false), there is no UI here to await for a UI or Main thread

Reply:
agree